web/src/main/scala/lucuma/seed/Main.scala

```diff
@@ -18,15 +19,10 @@ import scala.scalajs.js.annotation.JSExport
 import scala.scalajs.js.annotation.JSExportTopLevel
 
 @JSExportTopLevel("Main")
-object Main extends IOApp.Simple {
+object Main {
```

Comment:
We don't need the IOApp stuff anymore? I guess the `import cats.effect.unsafe.implicits._` takes care of setting up the runtime?

Reply:
This was recommended to avoid having to restart the runtime.
The trick to restart the runtime had to use structural typing which is not available in scala 3 anymore

Reply:
Cool.